ui/packages/components/src/utils/theme.ts

```diff
@@ -0,0 +1,12 @@
+//
+// This will futureproof us a bit if we switch to manual or system color scheme.
+// If an element ref is provided we look for a dark class in any parents.
+// This is useful for components that might have dark theme sections inside a general light theme
+export const colorScheme = (elementRef?: any): 'light' | 'dark' =>
```

Comment:
I think both colorScheme and isDark can have a `HTMLElement` type instead of `any`

Reply:
Yup, will do!